### PR TITLE
Added troubleshooting documentation

### PR DESCRIPTION
**Pull request by odinnordico**
Pull request

### What this PR does / why we need it
Added documentation of troubleshooting 

### Which issue(s) this PR fixes

Fixes #
N/A
### Describe testing done for PR
Links follow through web browser

### CODE REVIEW

**Review by rashedkvm — COMMENTED** [review]
Please do not merge this doc. We need to word-smith the content.

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    For example:
+    
+    ```bash
+    NAME                       READY   STATUS    RESTARTS   AGE
+    manager-5dbb5f584c-zvmg2   1/1     Running   0          11d
+    ```
+
+2. Gather the actual logs from the `Pod` using the `kubectl` command.
+   
+   `$ kubectl -n service-bindings logs manager-5dbb5f584c-zvmg2`
```
Comment (shashwathi) [inline docs/troubleshooting.md:18]: Update kubectl command to use labels
`kubectl -n service-bindings logs -l role=manager` . This removes the need to get Pod name

**Review by shashwathi — CHANGES_REQUESTED** [review]

File: docs/troubleshooting.md (hunk 1 above)
Comment (odinnordico) [reply docs/troubleshooting.md]: Thank you @shashwathi, good improvement

**Review by shashwathi — COMMENTED** [review]
looks good

**Review by shashwathi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ kapp deploy -a service-bindings -f <(ko resolve -f config)
 kapp delete -a service-bindings
 ```
 
+## Troubleshooting
+
+Service Bindings for Kubernetes is an implementation of the Service Binding Specification for Kubernetes, If any trouble shows up please refer to the troubleshooting guide [here](./docs/troubleshooting.md).
```
Comment (rashedkvm) [inline README.md:49]:
```suggestion
For basic troubleshooting Service Bindings, please see the troubleshooting guide [here](./docs/troubleshooting.md).
```

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Troubleshooting
+
+## Collecting logs from service binding manager
+
+Retrieve the logs from the service-bindings `Pod` in the `service-bindings` namespace using
```
Comment (rashedkvm) [inline docs/troubleshooting.md:5]:
```suggestion
Retrieve pod logs from the `manager` running in the `service-bindings` namespace. 
```

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+    `kubectl -n service-bindings logs -l role=manager`
```
Comment (rashedkvm) [inline docs/troubleshooting.md:7]:
```suggestion
    ```bash
     kubectl logs manager  -n service-bindings
     ```
```

Comment (odinnordico) [reply docs/troubleshooting.md]: Changed

File: README.md (hunk 2 above)
Comment (odinnordico) [reply README.md]: Changed

**Review by rashedkvm — APPROVED** [review]
lgtm - it is a good start for such doc.

**Pull request merged**